crates/starfish/config/src/parameters.rs

```diff
@@ -124,8 +128,8 @@ impl Parameters {
         Duration::from_millis(500)
     }
 
-    pub(crate) fn default_max_block_headers_per_fetch() -> usize {
-        // TODO: set some sensible value adjusted for Starfish
+    // Maximum number of block headers to fetch per commit sync request.
+    pub(crate) fn default_max_headers_per_commit_sync_fetch() -> usize {
```

Comment:
Are those values good for Starfish as well? You removed a TODO, and perhaps we should create an issue to review all Starfish parameters eventually.